README.md

```diff
@@ -48,9 +68,25 @@ or
 pip install git+https://github.com/The-AI-Alliance/gofannon.git
 ```
 
+## Communication Channels
+- **Disccord**: Real-time collaboration and support
```

Comment:
spelling, copy link from here
https://github.com/The-AI-Alliance/gofannon/blob/main/website/community/index.md?plain=1#L14
value of that liquid variable can be found here:
https://github.com/The-AI-Alliance/gofannon/blob/main/website/_config.yml#L8